$NetBSD: patch-ab,v 1.1 2005/03/16 21:01:58 rillig Exp $

No standard ever mentions "struct fd_set", so use the widely-known type
fd_set instead. Needed for Linux.

--- xjig.C.orig	Wed Jul 24 22:14:28 1996
+++ xjig.C	Wed Mar 16 21:55:16 2005
@@ -668,7 +668,7 @@ Pixmap	pixmap;
 			XSync(dpy,0);
 			while (!XPending(dpy)) {
 				struct timeval timeout;
-				struct fd_set readfds;
+				fd_set readfds;
 				int		nfds;
 	
 				FD_ZERO( &readfds );
